aced_submission/cli.py

```diff
@@ -22,11 +22,19 @@ def cli():
 cli.add_command(files)
 cli.add_command(fhir_store)
 
-@cli.command(name='schema-publish')
-@click.argument('dictionary_path', default='iceberg/schemas/gen3/aced.json')
-@click.option('--bucket', default="s3://aced-public", help="Bucket target", show_default=True)
-@click.option('--production', default=False, is_flag=True, show_default=True,
-              help="Write to aced.json, otherwise aced-test.json")
+
+@cli.command(name="schema-publish")
+@click.argument("dictionary_path", default="iceberg/schemas/gen3/aced.json")
```

Comment:
Don't see this iceberg/schemas/gen3/aced.json schema on https://github.com/bmeg/iceberg/tree/main/schemas. The directory structure/name of the schema might have changed. There is also another aws link schema that gets used in the etl-pod. 

It would make things a bit less confusing if a standard name for the simplified and for the graph schema was used and was consistent across etl-pod/submission/iceberg/iceberg-schema-tools/etc... 

aced.json seems a bit too vague

Reply:
Let's re-examine in next sprint.  I really didn't add any code to aced_submission, I just ran flake8 against it so tests would pass.  @matthewpeterkort